src/components/course/data/hooks.jsx

```diff
@@ -43,11 +42,11 @@ import {
   DISABLED_ENROLL_USER_MESSAGES,
   DISABLED_ENROLL_REASON_TYPES,
   ENTERPRISE_OFFER_SUBSIDY_TYPE,
-  DATE_FORMAT,
 } from './constants';
 import { pushEvent, EVENTS } from '../../../utils/optimizely';
 import { getExternalCourseEnrollmentUrl } from '../enrollment/utils';
 import { createExecutiveEducationFailureMessage } from '../../executive-education-2u/ExecutiveEducation2UError';
+import { getCourseStartDate } from '../../executive-education-2u/utils';
```

Comment:
If we're using the same `getCourseStartDate` function for both OCM and Exec Ed courses, the function should probably be defined in a general place not specific to Exec Ed as it is now. Perhaps moving this function definition to `src/components/course/data/utils.jsx` instead? Also keep in mind all the components/functions in the `executve-education-2u` module will soon be refactored/moved closer to the `course` module to be co-located with the new routes for Exec Ed (i.e., `/enroll` and `/enroll/complete`).